config.js

```diff
@@ -14,7 +14,9 @@ window.config = {
 		// donations over this amount will be highlighted in the list (null to ignore highlighting)
 		highlightThreshold: 100,
 		// the number of donations to return (1-100)
-		listLimit: 25
+		listLimit: 25,
+		// the number of top donors to return (1-100)
+		topDonorListLimit: 25
```

Comment:
@benjaminrclayton this should be in a separate grouping of `topDonors`-specific toggles: 
```
	// properties specific to donation-ticker
	donationTicker: {
		...
	},

	// properties specific to top-donors
	topDonors: {
		listLimit: 25
	}
```